app/views/common/_flash.html.erb

```diff
@@ -0,0 +1,6 @@
+<% flash.each do |type, message| %>
```

Comment:
Consider adding: `<% next unless bootstrap_class_for(type) %>`, otherwise this will trigger for all flashes and create the alert no matter what (sort of future proofing this code, because a flash could easily be used for something other than an alert)